beetle_battle.py

```diff
@@ -5,30 +5,53 @@
 import arcade
 
 # Constants
-SCREEN_WIDTH = 1024
+SCREEN_WIDTH = 1280
 SCREEN_HEIGHT = 720
 SCREEN_TITLE = "Welcome to Arcade"
 
+# Scaling constants
+BEETLE_SCALING = .65
+
 # Classes
 class BeetleBattle(arcade.Window):
     def __init__(self):
 
         # Call the parent class constructor
         super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
         
+        # Beetle sprites in this variable
+        self.beetle_list = None
+        self.beetle_sprite = None
```

Comment:
I was about to suggest that you could move `beetle_sprite` down to a local variable, but thinking about it more, we will need it to be an attribute because we will need to refer to it to add more beetles to `beetle_list`.

Perhaps we could instead call this `beetle_sprite_prototype` or even just `beetle_prototype` as this single sprite serves as a prototype for the other sprites, which then we could just use `beetle_sprites`. But we can definitely postpone that decision until we are deeper in using these lists. We'll likely have a better idea then.

Reply:
Ah, okay, this is why you linked me that Flyweight article eh